aea/aea.py

```diff
@@ -237,6 +238,23 @@ def react(self) -> None:
             counter += 1
             self._react_one()
 
+    @property
+    def active_connections(self) -> List[Connection]:
+        """Return list of active connections."""
+        connections = self.resources.get_all_connections()
+        if self._connection_ids is not None:
+            connections = [
+                c for c in connections if c.connection_id in self._connection_ids
+            ]
+        return connections
+
+    def _get_multiplexer_setup_options(self) -> Optional[Dict]:
```

Comment:
Contents of Dict need to be typed as well

Reply:
I think we should have something like
```
return {**super()._get_multiplexer_setup_options(), default_routing=self.context.default_routing, default_connection=self.context.default_connection}
```
here

Reply:
not sure, there can be a conflict, in this case, i want to set options explicitly

Reply:
I don't follow; this is to keep the Agent and AEA class symmetric here.

Reply:
Ok I agree now